src/components/ProposalDetailPanel.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { UsersIcon } from '@heroicons/react/24/solid';
+import {
+  Panel,
+  PanelBody,
+  PanelHeader,
+  PanelTag,
+  PanelTitle,
+  PanelTitleTags,
+  PanelTitleWrapper,
+} from './Panel';
+import { ProposalTable } from './ProposalTable';
+
+interface ProposalDetailPanelProps {
+  title: string;
+  author: string;
+  authorId: string | undefined;
+  version: number;
+  values: {
+    fieldName: string,
+    position: number,
+    value: string,
+  }[],
+}
+
+const ProposalDetailPanel = ({
+  title,
+  author,
+  authorId,
+  version,
+  values,
+}: ProposalDetailPanelProps) => (
+  <Panel>
+    <PanelHeader>
+      <PanelTitleWrapper>
+        <PanelTitle>
+          {title}
+        </PanelTitle>
+        <PanelTitleTags>
+          <PanelTag>
+            <UsersIcon className="icon" />
```

Comment:
@jim-mcgowan was not a fan of this users icon: "I'm not sure what that's trying to convey. Three people would suggest an organization to me."

There is a `user-group` in heroicons; I'll try that.

Reply:
I don't think this looks great:

![Screen Shot 2023-03-31 at 18 44 27](https://user-images.githubusercontent.com/1494855/229259819-9953693d-1a47-4fe0-9c6c-804c9e32fee4.png)

It's not just the alignment, although that certainly doesn't help. I think I'll omit the icon entirely, unless someone speaks up.

Reply:
I agree with both of you, in fact! I too wanted the three-person icon, but ended up not liking it because it looks vaguely like a paw print more than a crowd — at least, I couldn't unthink it after thinking it.

I don't think the big title needs an icon, so I also agree with where you landed. But, as noted in my other comment, I want to take a crack at this header altogether after understanding Jim's feedback better.

I did notice that you used the building library icon for the EIN. I feel like without the organization label also in the same tag, we can't use such an "organization as a building" metaphor. I'm experimenting with just a small badge reading "EIN", which no doubt we'll change once we replace EIN as the most visible organization ID:

<img width="278" alt="Organization header" src="https://user-images.githubusercontent.com/4731/229990470-56123cb7-49c5-4cc4-932f-e5e6525d31d7.png">

Reply:
Oh, that looks very good!